test/commands/new.test.ts

```diff
@@ -0,0 +1,12 @@
+import { expect, test } from '@oclif/test';
+
+describe('new', () => {
+  test
+    .stderr()
+    .stdout()
+    .command(['new', '--no-tty', '-n=specification.yaml'])
+    .it('runs new command', (ctx, done) => {
+      expect(ctx.stdout).to.contain('specification.yaml');
```

Comment:
```suggestion
      expect(ctx.stderr).to.equals('');
      expect(ctx.stdout).contains('specification.yaml');
```

This is working for me, you can try this once.

Reply:
Still showing error 😟😟

![image](https://user-images.githubusercontent.com/54466041/155838498-40b4911d-d86b-4c55-8d6d-481862e74a26.png)